stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/AddressElement.kt

```diff
@@ -67,9 +66,11 @@ open class AddressElement constructor(
     private val elementsRegistry = AddressElementUiRegistry(AddressSchemaRegistry)
 
     private val otherFields = countryElement.controller.rawFieldValue
-        .map { countryCode ->
+        .mapAsStateFlow { countryCode ->
             countryCode?.let {
-                phoneNumberElement.controller.countryDropdownController.onRawValueChange(it)
+                if (phoneNumberElement.controller.fieldValue.value.isBlank()) {
```

Comment:
Is this a behavior change?

Reply:
Yup. I initially thought this fix was to maintain behavior but this is not the case. The issue you pointed out with `flattenConcatAsStateFlow` was the reason for this issue.